azure-pipelines.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   timeoutInMinutes: 20
   cancelTimeoutInMinutes: 2
   pool:
-    name: 'Hosted VS2017'
+    name: 'Hosted macOS'

```

Comment:
Why are we doing this change?

Reply:
2017 was nto working due the pool issue, also i felt using non-windows agent also helps to catch all of other issues related to OS [if we have any].